selfdrive/boardd/panda.cc

```diff
@@ -17,12 +17,12 @@ void panda_set_power(bool power){
   err += gpio_init(GPIO_STM_RST_N, true);
   err += gpio_init(GPIO_STM_BOOT0, true);
 
-  err += gpio_set(GPIO_STM_RST_N, true);
+  err += gpio_set(GPIO_STM_RST_N, false);
```

Comment:
why are there panda changes in a UI PR?

Reply:
That was an oversight... My bad will fix.